Lint with flake8

<!--
The GitHub issue tracker is for bug reports and feature requests. General support can be found at
the following locations:

- Google group - https://groups.google.com/forum/#!forum/mainflux
- Gitter - https://gitter.im/mainflux/mainflux
-->


**ENHANCEMENT**
1. Describe the enhancement you are requesting. Enhancements include:
   - Linting with flake8
   - code refactor

2. Should-have.

```cmd

lib/groups.py:14:80: E501 line too long (101 > 79 characters)
lib/groups.py:17:80: E501 line too long (103 > 79 characters)
lib/groups.py:26:80: E501 line too long (100 > 79 characters)
lib/groups.py:29:80: E501 line too long (100 > 79 characters)
lib/groups.py:52:80: E501 line too long (104 > 79 characters)
lib/groups.py:59:80: E501 line too long (112 > 79 characters)
lib/groups.py:63:80: E501 line too long (103 > 79 characters)
lib/groups.py:68:80: E501 line too long (114 > 79 characters)
lib/groups.py:72:80: E501 line too long (104 > 79 characters)
lib/groups.py:78:80: E501 line too long (128 > 79 characters)
lib/groups.py:81:80: E501 line too long (103 > 79 characters)
lib/groups.py:87:80: E501 line too long (130 > 79 characters)
lib/groups.py:90:80: E501 line too long (105 > 79 characters)
lib/groups.py:95:80: E501 line too long (103 > 79 characters)
lib/groups.py:99:80: E501 line too long (103 > 79 characters)
lib/examples.py:29:25: E999 SyntaxError: invalid syntax
lib/channels.py:14:80: E501 line too long (105 > 79 characters)
lib/channels.py:17:80: E501 line too long (105 > 79 characters)
lib/channels.py:26:80: E501 line too long (111 > 79 characters)
lib/channels.py:29:80: E501 line too long (110 > 79 characters)
lib/channels.py:37:80: E501 line too long (100 > 79 characters)
lib/channels.py:40:80: E501 line too long (102 > 79 characters)
lib/channels.py:56:5: E303 too many blank lines (2)
lib/channels.py:64:80: E501 line too long (106 > 79 characters)
lib/channels.py:77:80: E501 line too long (111 > 79 characters)
lib/channels.py:84:80: E501 line too long (118 > 79 characters)
lib/channels.py:88:80: E501 line too long (105 > 79 characters)
lib/channels.py:93:80: E501 line too long (103 > 79 characters)
lib/channels.py:97:80: E501 line too long (105 > 79 characters)
lib/response.py:8:1: E303 too many blank lines (3)
lib/things.py:14:80: E501 line too long (101 > 79 characters)
lib/things.py:17:80: E501 line too long (103 > 79 characters)
lib/things.py:26:80: E501 line too long (107 > 79 characters)
lib/things.py:29:80: E501 line too long (108 > 79 characters)
lib/things.py:37:80: E501 line too long (100 > 79 characters)
lib/things.py:40:80: E501 line too long (100 > 79 characters)
lib/things.py:63:80: E501 line too long (104 > 79 characters)
lib/things.py:76:80: E501 line too long (111 > 79 characters)
lib/things.py:83:80: E501 line too long (112 > 79 characters)
lib/things.py:87:80: E501 line too long (103 > 79 characters)
lib/things.py:92:80: E501 line too long (103 > 79 characters)
lib/things.py:96:80: E501 line too long (103 > 79 characters)
lib/things.py:105:80: E501 line too long (104 > 79 characters)
lib/things.py:109:80: E501 line too long (104 > 79 characters)
lib/things.py:116:80: E501 line too long (133 > 79 characters)
lib/things.py:120:80: E501 line too long (107 > 79 characters)
lib/messages.py:15:80: E501 line too long (82 > 79 characters)
lib/messages.py:18:80: E501 line too long (103 > 79 characters)
lib/messages.py:28:80: E501 line too long (103 > 79 characters)
lib/users.py:18:80: E501 line too long (102 > 79 characters)
lib/users.py:30:80: E501 line too long (101 > 79 characters)
lib/users.py:38:80: E501 line too long (98 > 79 characters)
lib/users.py:41:80: E501 line too long (99 > 79 characters)
lib/users.py:48:80: E501 line too long (87 > 79 characters)
lib/users.py:52:80: E501 line too long (103 > 79 characters)
lib/users.py:59:80: E501 line too long (110 > 79 characters)
lib/users.py:63:80: E501 line too long (102 > 79 characters)
lib/users.py:72:80: E501 line too long (106 > 79 characters)
lib/users.py:76:80: E501 line too long (111 > 79 characters)

```

--- a/lib/channels.py
+++ b/lib/channels.py
@@ -11,10 +11,15 @@ class Channels:
     def create(self, channel, token):
         '''Creates channel entity in the database'''
         mf_resp = response.Response()
-        http_resp = requests.post(self.url + "/channels", json=channel, headers={"Authorization": token})
+        http_resp = requests.post(
+            self.url + "/channels",
+            json=channel,
+            headers={"Authorization": token}
+        )
         if http_resp.status_code != 201:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.channels["create"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.channels["create"], http_resp.status_code)
         else:
             location = http_resp.headers.get("location")
             mf_resp.value = location.split('/')[2]
@@ -23,10 +28,15 @@ class Channels:
     def create_bulk(self, channels, token):
         '''Creates multiple channels in a bulk'''
         mf_resp = response.Response()
-        http_resp = requests.post(self.url + "/channels/bulk", json=channels, headers={"Authorization": token})
+        http_resp = requests.post(
+            self.url + "/channels/bulk",
+            json=channels,
+            headers={"Authorization": token}
+        )
         if http_resp.status_code != 201:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.channels["create_bulk"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.channels["create_bulk"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
@@ -34,10 +44,12 @@ class Channels:
     def get(self, chanID, token):
         '''Gets a channel entity for a logged-in user'''
         mf_resp = response.Response()
-        http_resp = requests.get(self.url + "/channels/" + chanID, headers={"Authorization": token})
+        http_resp = requests.get(
+            self.url + "/channels/" + chanID, headers={"Authorization": token})
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.channels["get"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.channels["get"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
@@ -52,7 +64,6 @@ class Channels:
                 query += "{}={}&".format(pt, params[pt])
         return query
 
-
     def get_all(self, token, query_params=None):
         '''Gets all channels from database'''
         query = self.construct_query(query_params)
@@ -61,7 +72,8 @@ class Channels:
         http_resp = requests.get(url, headers={"Authorization": token})
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.channels["get_all"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.channels["get_all"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
@@ -74,25 +86,33 @@ class Channels:
         http_resp = requests.get(url, headers={"Authorization": token})
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.channels["get_by_thing"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.channels["get_by_thing"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
 
     def update(self, channel_id, token, channel):
         '''Updates channel entity'''
-        http_resp = requests.put(self.url + "/channels/" + channel_id, json=channel, headers={"Authorization": token})
+        http_resp = requests.put(
+            self.url + "/channels/" + channel_id,
+            json=channel,
+            headers={"Authorization": token}
+        )
         mf_resp = response.Response()
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.channels["update"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.channels["update"], http_resp.status_code)
         return mf_resp
 
     def delete(self, chanID, token):
         '''Deletes a channel entity from database'''
-        http_resp = requests.delete(self.url + "/channels/" + chanID, headers={"Authorization": token})
+        http_resp = requests.delete(
+            self.url + "/channels/" + chanID, headers={"Authorization": token})
         mf_resp = response.Response()
         if http_resp.status_code != 204:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.channels["delete"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.channels["delete"], http_resp.status_code)
         return mf_resp

--- a/lib/groups.py
+++ b/lib/groups.py
@@ -11,10 +11,12 @@ class Groups:
     def create(self, group, token):
         '''Creates group entity in the database'''
         mf_resp = response.Response()
-        http_resp = requests.post(self.url + "/groups", json=group, headers={"Authorization": token})
+        http_resp = requests.post(
+            self.url + "/groups", json=group, headers={"Authorization": token})
         if http_resp.status_code != 201:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.groups["create"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.groups["create"], http_resp.status_code)
         else:
             location = http_resp.headers.get("location")
             mf_resp.value = location.split('/')[2]
@@ -23,10 +25,12 @@ class Groups:
     def get(self, group_id, token):
         '''Gets a group entity'''
         mf_resp = response.Response()
-        http_resp = requests.get(self.url + "/groups/" + group_id, headers={"Authorization": token})
+        http_resp = requests.get(
+            self.url + "/groups/" + group_id, headers={"Authorization": token})
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.groups["get"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.groups["get"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
@@ -49,52 +53,75 @@ class Groups:
         http_resp = requests.get(url, headers={"Authorization": token})
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.groups["get_all"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.groups["get_all"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
 
     def update(self, group_id, token, group):
         '''Updates group entity'''
-        http_resp = requests.put(self.url + "/groups/" + group_id, json=group, headers={"Authorization": token})
+        http_resp = requests.put(
+            self.url + "/groups/" + group_id,
+            json=group,
+            headers={"Authorization": token}
+        )
         mf_resp = response.Response()
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.groups["update"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.groups["update"], http_resp.status_code)
         return mf_resp
 
     def members(self, group_id, token):
         '''Get list of members ID's from group'''
-        http_resp = requests.post(self.url + "/groups/" + group_id + "/members", headers={"Authorization": token})
+        http_resp = requests.post(
+            self.url + "/groups/" + group_id + "/members",
+            headers={"Authorization": token}
+        )
         mf_resp = response.Response()
         if http_resp.status_code != 204:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.groups["members"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.groups["members"], http_resp.status_code)
         return mf_resp
 
     def assign(self, group_id, token, members):
         '''Assign'''
         mf_resp = response.Response()
-        http_resp = requests.post(self.url + "/groups/" + group_id + "/members", headers={"Authorization": token}, json=members)
+        http_resp = requests.post(self.url + "/groups/" + group_id +
+                                  "/members",
+                                  headers={"Authorization": token},
+                                  json=members
+                                  )
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.groups["assign"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.groups["assign"], http_resp.status_code)
         return mf_resp
 
     def unassign(self, group_id, token, members):
         '''Assign'''
         mf_resp = response.Response()
-        http_resp = requests.delete(self.url + "/groups/" + group_id + "/members", headers={"Authorization": token}, json=members)
+        http_resp = requests.delete(self.url + "/groups/" + group_id +
+                                    "/members",
+                                    headers={"Authorization": token},
+                                    json=members)
         if http_resp.status_code != 204:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.groups["unassign"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.groups["unassign"], http_resp.status_code)
         return mf_resp
 
     def delete(self, group_id, token):
         '''Deletes a group entity from database'''
-        http_resp = requests.delete(self.url + "/groups/" + group_id, headers={"Authorization": token})
+        http_resp = requests.delete(
+            self.url + "/groups/" + group_id,
+            headers={"Authorization": token}
+        )
         mf_resp = response.Response()
         if http_resp.status_code != 204:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.groups["delete"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.groups["delete"], http_resp.status_code)
         return mf_resp

--- a/lib/messages.py
+++ b/lib/messages.py
@@ -12,10 +12,12 @@ class Messages:
         '''Sends message via HTTP protocol'''
         url = self.url + "/http/channels/" + channel_id + '/messages'
         mf_resp = response.Response()
-        http_resp = requests.post(url, json=msg, headers={"Authorization": token})
+        http_resp = requests.post(url, json=msg, headers={
+                                  "Authorization": token})
         if http_resp.status_code != 202:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.messages["send"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.messages["send"], http_resp.status_code)
         return mf_resp
 
     def read(self, channel_id, token):
@@ -25,7 +27,8 @@ class Messages:
         http_resp = requests.get(url, headers={"Authorization": token})
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.messages["read"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.messages["read"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp

--- a/lib/response.py
+++ b/lib/response.py
@@ -4,7 +4,6 @@ class Error():
         self.message = message
 
 
-
 class Response(object):
     def __init__(self, value=None, error_status=0, error_message=""):
         self.error = Error(error_status, error_message)

--- a/lib/things.py
+++ b/lib/things.py
@@ -11,10 +11,12 @@ class Things:
     def create(self, thing, token):
         '''Creates thing entity in the database'''
         mf_resp = response.Response()
-        http_resp = requests.post(self.url + "/things", json=thing, headers={"Authorization": token})
+        http_resp = requests.post(
+            self.url + "/things", json=thing, headers={"Authorization": token})
         if http_resp.status_code != 201:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.things["create"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.things["create"], http_resp.status_code)
         else:
             location = http_resp.headers.get("location")
             mf_resp.value = location.split('/')[2]
@@ -23,10 +25,15 @@ class Things:
     def create_bulk(self, things, token):
         '''Creates multiple things in a bulk'''
         mf_resp = response.Response()
-        http_resp = requests.post(self.url + "/things/bulk", json=things, headers={"Authorization": token})
+        http_resp = requests.post(
+            self.url + "/things/bulk",
+            json=things,
+            headers={"Authorization": token}
+        )
         if http_resp.status_code != 201:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.things["create_bulk"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.things["create_bulk"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
@@ -34,10 +41,12 @@ class Things:
     def get(self, thing_id, token):
         '''Gets a thing entity for a logged-in user'''
         mf_resp = response.Response()
-        http_resp = requests.get(self.url + "/things/" + thing_id, headers={"Authorization": token})
+        http_resp = requests.get(
+            self.url + "/things/" + thing_id, headers={"Authorization": token})
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.things["get"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.things["get"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
@@ -60,7 +69,8 @@ class Things:
         http_resp = requests.get(url, headers={"Authorization": token})
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.things["get_all"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.things["get_all"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
@@ -73,49 +83,66 @@ class Things:
         http_resp = requests.get(url, headers={"Authorization": token})
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.things["get_by_channel"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.things["get_by_channel"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
 
     def update(self, thing_id, token, thing):
         '''Updates thing entity'''
-        http_resp = requests.put(self.url + "/things/" + thing_id, json=thing, headers={"Authorization": token})
+        http_resp = requests.put(
+            self.url + "/things/" + thing_id,
+            json=thing,
+            headers={"Authorization": token}
+        )
         mf_resp = response.Response()
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.things["update"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.things["update"], http_resp.status_code)
         return mf_resp
 
     def delete(self, thing_id, token):
         '''Deletes a thing entity from database'''
-        http_resp = requests.delete(self.url + "/things/" + thing_id, headers={"Authorization": token})
+        http_resp = requests.delete(
+            self.url + "/things/" + thing_id, headers={"Authorization": token})
         mf_resp = response.Response()
         if http_resp.status_code != 204:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.things["delete"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.things["delete"], http_resp.status_code)
         return mf_resp
 
     def connect(self, channel_ids, thing_ids, token):
         '''Connects thing and channel'''
         payload = {
-          "channel_ids": channel_ids,
-          "thing_ids": thing_ids,
+            "channel_ids": channel_ids,
+            "thing_ids": thing_ids,
         }
-        http_resp = requests.post(self.url + "/connect", headers={"Authorization": token}, json=payload)
+        http_resp = requests.post(
+            self.url + "/connect",
+            headers={"Authorization": token},
+            json=payload
+        )
         mf_resp = response.Response()
         if http_resp.status_code != 201:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.things["connect"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.things["connect"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
 
     def disconnect(self, channel_ids, thing_ids, token):
         '''Disconnect thing and channel'''
-        http_resp = requests.delete(self.url + "/channels/" + channel_ids + "/things/" + thing_ids, headers={"Authorization": token})
+        http_resp = requests.delete(self.url + "/channels/" + channel_ids +
+                                    "/things/" + thing_ids,
+                                    headers={"Authorization": token}
+                                    )
         mf_resp = response.Response()
         if http_resp.status_code != 204:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.things["disconnect"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.things["disconnect"], http_resp.status_code)
         return mf_resp

--- a/lib/users.py
+++ b/lib/users.py
@@ -15,7 +15,8 @@ class Users:
         http_resp = requests.post(self.url + "/users", json=user)
         if http_resp.status_code != 201:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.users["create"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.users["create"], http_resp.status_code)
         else:
             location = http_resp.headers.get("location")
             mf_resp.value = location.split('/')[2]
@@ -27,7 +28,8 @@ class Users:
         http_resp = requests.post(self.url + "/tokens", json=user)
         if http_resp.status_code != 201:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.users["login"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.users["login"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()["token"]
         return mf_resp
@@ -35,43 +37,57 @@ class Users:
     def get(self, user_id, token):
         '''Gets a user entity for a logged-in user'''
         mf_resp = response.Response()
-        http_resp = requests.get(self.url + "/users/" + user_id, headers={"Authorization": token})
+        http_resp = requests.get(
+            self.url + "/users/" + user_id, headers={"Authorization": token})
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.users["get"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.users["get"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
 
     def get_all(self, token):
         '''Gets all users from database'''
-        http_resp = requests.get(self.url + "/users", headers={"Authorization": token})
+        http_resp = requests.get(
+            self.url + "/users", headers={"Authorization": token})
         mf_resp = response.Response()
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.users["get_all"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.users["get_all"], http_resp.status_code)
         else:
             mf_resp.value = http_resp.json()
         return mf_resp
 
     def update(self, user, token):
         '''Updates user entity'''
-        http_resp = requests.put(self.url + "/users", headers={"Authorization": token}, data=json.dumps(user))
+        http_resp = requests.put(
+            self.url + "/users",
+            headers={"Authorization": token},
+            data=json.dumps(user)
+        )
         mf_resp = response.Response()
         if http_resp.status_code != 200:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.users["update"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.users["update"], http_resp.status_code)
         return mf_resp
 
     def update_password(self, old_password, password, token):
         '''Changes user password'''
         payload = {
-          "old_password": old_password,
-          "password": password
+            "old_password": old_password,
+            "password": password
         }
-        http_resp = requests.patch(self.url + "/password", headers={"Authorization": token}, json=payload)
+        http_resp = requests.patch(
+            self.url + "/password",
+            headers={"Authorization": token},
+            json=payload
+        )
         mf_resp = response.Response()
         if http_resp.status_code != 201:
             mf_resp.error.status = 1
-            mf_resp.error.message = errors.handle_error(errors.users["update_password"], http_resp.status_code)
+            mf_resp.error.message = errors.handle_error(
+                errors.users["update_password"], http_resp.status_code)
         return mf_resp

